### ISSUE TITLE
zsh command completions

### ISSUE THREAD

**Issue by jRimbault** [subnixr/minimal#7]
It seems `minimal` breaks zsh "clever history" ?  
```sh
bindkey '\e[A' history-search-backward
bindkey '\e[B' history-search-forward
```

**Comment by subnixr** [comment]
Strange, minimal just binds `^M` to magic enter (and it's optional). 

Also, I'm not able to reproduce the problem (running `zsh -df` and hitting `alt + A` doesn't trigger anything).

Can you provide an example and configuraiton files?

**Comment by jRimbault** [comment]
`'\e[A'` and `'\e[A'` are the up and down arrow keys.  
I went though the script and indeed there's isn't anything obvious which would mess with that.

The expected behavior is: when typing `cd D` and then `up` it should look up the previous command beginning by that, instead it just goes to the most recent command.

I fixed it on my end by just appending the two `bindkey` commands to your theme.
I'll look into it more but, after replicating on a blank slate, it seems it's an issue on my system alone and independent from your theme.

Sorry for the inconvenience.
